src/magic.cpp

```diff
@@ -352,6 +356,7 @@ spell::spell( const spell_type *sp, int xp )
 {
     type = sp;
     experience = xp;
+    obj_name = "";
```

Comment:
This is a pointless statement. `obj_name` has just been created, it **is** already empty.

Reply:
changed (i thought strings wouldn't initialize like char* does), due to another commit a joint commit was impossible unfortunately.